.github/workflows/ci-cd.yml

```diff
@@ -93,20 +93,19 @@ jobs:
     needs: [lint, test-app]
     runs-on: ubuntu-latest
     timeout-minutes: 5
-    if: success() && github.ref == 'refs/heads/master' && github.event_name == 'push'
+    # Only run on pushes to main branch that aren't from the cron workflow
```

Comment:
I updated the steps to match the template from https://gist.github.com/ijlee2/1c864ebe96a55f239e80800829ef0bf4